### ISSUE TITLE
How does this work?

### ISSUE THREAD

**Issue by shibekin69** [RechInformatica/rech-open-this#1]
Hi,

I'm trying to get your add-on to work but it's not working out for me. My set up is something like this:

Project Open Has The Following Tree:

  Project/Path/To/Interested/FileIWantToOpen.json

I have a file that's opened:

  Project/Path/To/Interested/AnotherFile.json

And In this file, I have a text that's like this:

  ...
  some script line FileIWantToOpen
  ...

I try and put the text cursor under FileIWantToOpen, do right click and select the "Open This" command, but nothing's happening.

If I rename the line to:

  ...
  some script line FileIWantToOpen.json
  ...

If I try to do the same thing, it tells me it can't find the file FileIWantToOpen.json.

In the plugin's setting, I have the following:
-Search in opened project: Checked
-Path(s) for looking up requested files: Empty
-Extensions that are optional in filenames: .js;.json

**Comment by shibekin69** [comment]
Ah, it needs me to add the following to work:

Path(s) for looking up requested files: 
Project/Path/To/Interested/

Extensions that are optional in filenames:
.js;.json

It seems it doesn't recursively search the Project Directory for FileIWantToOpen with the extensions specified. Can this feature be requested?

**Comment by Jouwee** [comment]
Hi,

The package should look for the files in the project path recursively if the "Search files in opened project" setting is enabled (Which it is by default).

About it not working without the .json extension, althought it is optional, the package only checks tokens that look like filenames for performance reasons, such as filename.extension or "filename" (in quotes). I can maybe create a new setting for specifying a limit of tokens to check even if they don't look like filenames, so it would work on the example you sent and performance would not be a problem.

I will look into both of those situations

**Comment by shibekin69** [comment]
Yeah, maybe a separate search command or grabbing only the highlighted text might work?
